Kernel/run

```diff
@@ -58,6 +58,13 @@ elif [ "$1" = "qtext" ]; then
         -device e1000 \
         -kernel kernel \
         -append "${SERENITY_KERNEL_CMDLINE} text_debug"
+elif [ "$1" = "qpio" ]; then
+    # ./run: qemu with serenity (IDE controller with PIO mode)
+    $SERENITY_QEMU_BIN \
+        $SERENITY_COMMON_QEMU_ARGS \
+        -device e1000 \
+        -kernel kernel \
+        -append "${SERENITY_KERNEL_CMDLINE} force_pio"
```

Comment:
Maybe we should simply append any extra arguments to `./run` as kernel command line arguments.
It's a bit silly that we have to keep coming up with more `qsomething` variants :)

Reply:
> Maybe we should simply append any extra arguments to `./run` as kernel command line arguments.
> It's a bit silly that we have to keep coming up with more `qsomething` variants :)
Although I think we are fine with delegation of `qsomething` variants for now, I agree and I changed it so now we can add boot parameters via the script commandline.